townhall/myapi/tests.py

```diff
@@ -21,6 +21,13 @@ def test_get_volunteer(self):
         assert volunteer_1.age == 11
         assert volunteer_1.email == "zamorak.red@gmail.com"
 
+    def test_delete_volunteer(self):
+        townhall_services.VolunteerServices.delete_volunteer(id=1)
```

Comment:
before calloimg delete_volunteer, it would be good to call get_volunter and assert` is not None`